executor/executor_test.go

```diff
@@ -1951,58 +1953,79 @@ func (s *testSuite) TestIssue4024(c *C) {
 	tk.MustQuery("select * from test2.t").Check(testkit.Rows("2"))
 }
 
+const (
+	checkRequestOff          = 0
+	checkRequestPriority     = 1
+	checkRequestNotFillCache = 2
+)
+
 type checkRequestClient struct {
 	tikv.Client
-	priority pb.CommandPri
-	mu       struct {
+	priority     pb.CommandPri
+	notFillCache bool
+	mu           struct {
 		sync.RWMutex
-		turnOn bool
+		checkFlags uint32
 	}
 }
 
 func (c *checkRequestClient) SendReq(ctx goctx.Context, addr string, req *tikvrpc.Request) (*tikvrpc.Response, error) {
 	resp, err := c.Client.SendReq(ctx, addr, req)
 	c.mu.RLock()
-	turnOn := c.mu.turnOn
+	checkFlags := c.mu.checkFlags
 	c.mu.RUnlock()
-	if turnOn {
+	if checkFlags == checkRequestPriority {
```

Comment:
flag value should use bit operation like `checkFlags & checkRequestPriority > 0`

Reply:
I don't need to check both of them now, so this is unnecessary.